commerce-notification-service/src/main/java/com/liferay/commerce/notification/service/impl/CommerceNotificationQueueEntryServiceImpl.java

```diff
@@ -81,6 +101,39 @@ public int getCommerceNotificationQueueEntriesCount(long groupId)
 			getCommerceNotificationQueueEntriesCount(groupId);
 	}
 
+	@Override
+	public int getCommerceNotificationQueueEntriesCount(
```

Comment:
@Commando18 We don't need those methods, For example in the Order Admin UI, if you can see the order and you can see that portlet than you can see this.